.travis.yml

```diff
@@ -21,7 +21,8 @@ before_script:
   - ./configure --enable-ccache
 script:
   - make tidy
-  - make rustc-stage1 -j4
+  - make -j4
```

Comment:
Running `make check` in parallel has historically caused... exotic problems. Tests already run in parallel regardless (though test groups won't).